src/pages/Music.tsx

```diff
@@ -1,22 +1,198 @@
+    // dummy variable to represent the total time of the song in seconds
+    const seconds = 167;
+
+    /**
+     * useStates to manage toggling the play/pause and track
+     * current time of the song
+     */
+    const [isPlaying, setPlayingState] = useState<boolean>();
+    const [secPassed, setSecPassed] = useState(0);
+
+    /**
+     * useEffect hook to synchronize the time with the bar
+     * and acts as a timer for the current song
+     */
+    useEffect(() => {
+        // exit early when we reach 0
+        if (secPassed == seconds || !isPlaying) return;
+
+        // save intervalId to clear the interval when the
+        // component re-renders
+        const intervalId = setInterval(() => {
+            setSecPassed(secPassed + 1);
+        }, 1000);
+        // clear interval on re-render to avoid memory leaks
+        return () => clearInterval(intervalId);
+        // add timeLeft as a dependency to re-rerun the effect
+        // when we update it
+    }, [isPlaying, secPassed]);
+
+    /**
+     * dummy variables for the current song
+     */
+    const title = 'Sunset Blvd';
+    const artist = 'Kam Prada';
+    const album = 'Sunset Blvd';
+
+    /**
+     * function that turns integer number of seconds into a string
+     * formatted as mm:ss or m:ss
+     * @param sec integer number of seconds to turn into a string
+     */
+    function getSecondsAsDigitalClock(sec: number) {
+        const secNum = parseInt(sec.toString(), 10);
+
+        // Note: hours included just in case but hrs not displayed (assumed to be 0)
+        const hours = Math.floor(secNum / 3600);
+        const minutes = Math.floor((secNum - hours * 3600) / 60);
+        const seconds = secNum - hours * 3600 - minutes * 60;
+        let minutesString = '';
+        let secondsString = '';
+        minutesString = minutes.toString();
+        secondsString = seconds < 10 ? '0' + seconds : seconds.toString();
+        return minutesString + ':' + secondsString;
+    }
+
     return (
         <IonPage>
             <IonHeader>
-                <IonToolbar>
+                {/**
+                 * Purple toolbar with the title of the page: music
+                 */}
+                <IonToolbar color="#885c94" className="MusicToolBar">
                     <IonTitle>Music</IonTitle>
                 </IonToolbar>
             </IonHeader>
+
             <IonContent fullscreen>
                 <IonHeader collapse="condense">
                     <IonToolbar>
                         <IonTitle size="large">Music</IonTitle>
                     </IonToolbar>
                 </IonHeader>
-                <ExploreContainer name="Music Page" />
+
+                {/**
+                 * Create a grid to separate the album cover section (top left) from
+                 * the details/button section (top right) and the timebar section (bottom).
+                 */}
+                <IonGrid className="MusicGrid">
+                    {/**
```

Comment:
Huge props for the inline docs on the UI components. @b4yc @thomaskahessay @brooke-k we should try to follow this approach when documenting our code as well.